### PR TITLE
Enable static configuration vi runtime properties file only

### PR DESCRIPTION
**Pull request by b-slim**
In This PR i have removed all the code related to announcing namespace via the coordinator.
I have created a separate class that reads the namespace configuration.

### CODE REVIEW

**Comment by drcrallen** [comment]
Thanks for the contrib @b-slim , can you please rebase this off e0ae7e9f86192b76246c48bf25696cfc249b05f6 and squash your commits?

**Comment by b-slim** [comment]
@drcrallen Done

File: extensions/namespace-lookup/src/main/java/io/druid/server/initialization/NamespaceLookupStaticConfig.java [hunk 1]
```diff
@@ -0,0 +1,19 @@
+package io.druid.server.initialization;
```
Comment (drcrallen) [inline extensions/namespace-lookup/src/main/java/io/druid/server/initialization/NamespaceLookupStaticConfig.java]: license header

**Comment by drcrallen** [comment]
@b-slim : Can you make your commit look something more like this:

https://github.com/metamx/druid/commit/65b77be6c1b69a807f5997d2035cf0d12746f97d ?

**Comment by drcrallen** [comment]
oops, have too much deleted in NamespacedExtractionModule

**Comment by drcrallen** [comment]
I cherry-picked the commit into the QTL branch

**Comment by b-slim** [comment]
@drcrallen which branch are we talking about ?

**Pull request closed without merging**